app/institutions/dashboard/-components/projects-panel/component.ts

```diff
@@ -0,0 +1,39 @@
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+import InstitutionModel from 'ember-osf-web/models/institution';
+
+export default class ProjectsPanel extends Component {
+    institution!: InstitutionModel;
+
+    chartOptions = {
+        aspectRatio: 1,
+        legend: {
+            display: false,
+        },
+    };
+
+    @computed('institution.statSummary.{numPrivateProjects,numPublicProjects}')
+    get numProjects() {
+        if (!this.institution) {
+            return 0;
+        }
+        return this.institution.statSummary.numPrivateProjects + this.institution.statSummary.numPublicProjects;
```

Comment:
These longer lines get a little hard to read, consider:
```suggestion
        const { numPrivateProjects, numPublicProjects } = this.institution.statSummary;
        return numPrivateProjects + numPublicProjects;
```